packages/runtime/container-runtime/src/blobManager.ts

```diff
@@ -619,14 +603,14 @@ export class BlobManager extends TypedEventEmitter<IBlobManagerEvents> {
 					entry.status = PendingBlobStatus.OnlinePendingOp;
 				}
 			} else if (entry.status === PendingBlobStatus.OfflinePendingUpload) {
-				// We already submitted a BlobAttach op for this blob when it was transitioned to offline flow
+				// We already submitted a BlobAttach op for this blob when it was shutdown
 				entry.status = PendingBlobStatus.OfflinePendingOp;
```

Comment:
i expected the offline states to go away, or at least be unified? are they still needed? Could the names be improved?

Reply:
Yes, I'm working on that. They can be changed as "stashed" or something like that.

Reply:
in general i thought this whole class wouldn't be concerned with online or offline anymore, it would only be about shutdown and loading with local state. we should use this as an opportuity to clean up the code, such that is is clearer in the future.

Reply:
we can also consider doing clean up in multiple PRs if creates churn in things like tests.